BrainPortal/app/controllers/userfiles_controller.rb

```diff
@@ -86,16 +85,19 @@ def index #:nodoc:
 
     # Prepare the Pagination object
     @scope.pagination ||= Scope::Pagination.from_hash({ :per_page => 25 })
-    @scope.pagination.per_page = 999_999_999 unless
-      [:html, :js].include?(request.format.to_sym)
     @current_offset = (@scope.pagination.page - 1) * @scope.pagination.per_page
+    api_request     = ! [:html, :js].include?(request.format.to_sym)
+
+    # Special case; only userfile IDs are required (API request)
+    if params[:ids_only] && api_request
+      @userfiles = @view_scope.raw_first_column(:id)
```

Comment:
I have a very slight doubt that this is not an issue. I'll test if it indeed requires a fully quoted column name later today.